test/relations.test.js

```diff
@@ -34,9 +34,16 @@ describe('relations', function() {
     db = getSchema();
   });
 
+  after(function(done) {
+     // disconnect from this db to avoid too many connection error
+     // due to multiple instance of connection pool
+    db.disconnect();
+    done();
+  });
+
   describe('hasMany', function() {
     before(function(done) {
-      db = getSchema();
+      // db = getSchema();
```

Comment:
@raymondfeng hi, thanks for this info.
so it should work with one single getSchema() per file without a db.disconnect(), let's try this.

my successful test was actually removing all extraneous getSchema() and adding on db.disconnect(), if getSchema()  returns a singleton, then it the extraneous ones should be harmless... weird.